plugin/python_dependency_usage/python_dependency_usage.go

```diff
@@ -0,0 +1,191 @@
+package python_dependency_usage
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/safedep/code/core"
+	sitter "github.com/smacker/go-tree-sitter"
+)
+
+type DependencyUsagePlugin struct{}
+
+// Verify contract
+var _ core.TreePlugin = (*DependencyUsagePlugin)(nil)
+
+func (p *DependencyUsagePlugin) Name() string {
+	return "DependencyUsagePlugin"
+}
+
+func (p *DependencyUsagePlugin) AnalyzeTree(ctx context.Context, tree core.ParseTree) error {
+	lang, err := tree.Language()
+	if err != nil {
+		return fmt.Errorf("failed to get language: %w", err)
+	}
+
+	if lang.Meta().Name != "python" {
+		return nil
+	}
+
+	file, err := tree.File()
+	if err != nil {
+		return fmt.Errorf("failed to get file: %w", err)
+	}
+
+	imports, err := lang.Resolvers().ResolveImports(tree)
+	if err != nil {
+		return fmt.Errorf("failed to resolve imports: %w", err)
+	}
+
+	fmt.Println()
```

Comment:
You can use `log.Debugf` from `github.com/safedep/dry/log` package. It will only print when you set `APP_LOG_LEVEL=debug` as environment variable

Reply:
ack.
since i was experimenting, I have lot of unnecessary logs too, will remove those